documentation/modules/exploit/linux/local/blueman_set_dhcp_handler_dbus_priv_esc.md

```diff
@@ -0,0 +1,84 @@
+## Description
+
+  This module attempts to gain root privileges by exploiting a Python
+  code injection vulnerability in blueman versions prior to 2.0.3.
+
+  The `org.blueman.Mechanism.EnableNetwork` D-Bus interface exposes the
+  `set_dhcp_handler` function which uses user input in a call to `eval`,
+  without sanitization, resulting in arbitrary code execution as root.
+
+  This module has been tested successfully with blueman version 1.23
+  on Debian 8 Jessie (x64).
+
+
+## Vulnerable Application
+
+  This module has been tested successfully with:
+
+  * blueman version 1.23 on Debian 8 Jessie (x64)
+
+  Old versions of the `blueman` package are available in [Debian snapshots](https://snapshot.debian.org/).
+
+  The following `/etc/apt/sources.list` configuration will allow a vulnerable
+  version of the `blueman` package to be installed:
+ 
+  ```
+  deb [check-valid-until=no] http://snapshot.debian.org/archive/debian/20140827T042507Z/ jessie main
+  deb-src [check-valid-until=no] http://snapshot.debian.org/archive/debian/20140827T042507Z/ jessie main
+  ```
+
+  The package can be installed with `apt-get install blueman`
```

Comment:
Thanks for adding these simple 'how to install' instructions!

Reply:
Updated to mention `apt-get -o Acquire::Check-Valid-Until=false update`